src/test/java/org/broadinstitute/hellbender/tools/FixCallSetSampleOrderingIntegrationTest.java

```diff
@@ -0,0 +1,116 @@
+import htsjdk.variant.vcf.*;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.tools.genomicsdb.GenomicsDBImport;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.variant.GATKVariantContextUtils;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class FixCallSetSampleOrderingIntegrationTest extends CommandLineProgramTest{
+
+    private static final String SAMPLE_NAME_KEY = "SN";
+
+    private static File writeManyVCFs(int howMany) throws IOException {
+        Map<String,File> nameToFileMap = new LinkedHashMap<>();
+        final String contig = "chr20";
+        final SAMSequenceDictionary dict = new SAMSequenceDictionary(
+                Collections.singletonList(new SAMSequenceRecord(contig, 64444167)));
+
+        final VCFFormatHeaderLine formatField = new VCFFormatHeaderLine(SAMPLE_NAME_KEY, 1, VCFHeaderLineType.String,
+                                                                        "the name of the sample this genotype came from");
+        final Set<VCFHeaderLine> headerLines = new HashSet<>();
+        headerLines.add(formatField);
+        headerLines.add(VCFStandardHeaderLines.getFormatLine("GT"));
+
+        for( int i = 0; i < howMany ; i++) {
+            final File out = File.createTempFile("tiny_", ".vcf");
+
+            try (final VariantContextWriter writer = GATKVariantContextUtils.createVCFWriter(out, dict, false, Options.INDEX_ON_THE_FLY)) {
+                final String sampleName = "Sample_" + String.valueOf(i);
+
+                final VCFHeader vcfHeader = new VCFHeader(headerLines, Collections.singleton(sampleName));
+                vcfHeader.setSequenceDictionary(dict);
+                writer.writeHeader(vcfHeader);
+                final Allele Aref = Allele.create("A", true);
+                final Allele C = Allele.create("C");
+                final List<Allele> alleles = Arrays.asList(Aref, C);
+                final VariantContext variant = new VariantContextBuilder("invented", contig, 100, 100, alleles)
+                        .genotypes(new GenotypeBuilder(sampleName, alleles).attribute(SAMPLE_NAME_KEY, sampleName).make())
+                        .make();
+                writer.add(variant);
+                nameToFileMap.put(sampleName, out);
+            }
+        }
+
+        final File mapping = IOUtils.writeTempFile(
+                nameToFileMap.entrySet().stream()
+                        .map(pair -> pair.getKey() + "\t" + pair.getValue())
+                        .sorted(Comparator.reverseOrder())
+                        .collect(Collectors.joining("\n")), "mapping", ".sample_map");
+     //   mapping.deleteOnExit();
```

Comment:
Either remove this or explain why it's commented out.

Reply:
deleted